assets/src/containers/ResourcesAccessed.js

```diff
@@ -99,41 +106,28 @@ function ResourcesAccessed (props) {
     }
   }
 
-  const changeDefaultSetting = (event) => {
-    const didUserChecked = event.target.checked
-
-    setDefaultCheckedState(didUserChecked)
-    setDefaultLabel(didUserChecked ? currentSetting : rememberSetting)
-
-    if (didUserChecked) {
-      // Django rejects PUT/DELETE/POST calls with out CSRF token.
-      const csrfToken = Cookie.get('csrftoken')
-      const body = { resource: gradeRangeFilter }
-      const dataURL = `/api/v1/courses/${courseId}/set_user_default_selection`
-
-      defaultFetchOptions.headers['X-CSRFToken'] = csrfToken
-      defaultFetchOptions['method'] = 'PUT'
-      defaultFetchOptions['body'] = JSON.stringify(body)
+  useEffect(() => {
+    // if user setting is different from current grade filter, show label for remembering setting
+    if (userSavedFilterSetting !== resourceGradeFilter) {
+      setSaveLabel(rememberSetting)
+    } else if (savingError) {
+      setSaveLabel(settingNotUpdated)
+    } else {
+      setSaveLabel(currentSetting)
+    }
+  })
 
-      fetch(dataURL, defaultFetchOptions)
-        .then(handleError)
-        .then(res => res.json())
-        .then(data => {
-          const res = data.default
-          if (res === 'success') {
-            setGradeRangeFilter(gradeRangeFilter)
-            setDefaultCheckedState(true)
-            setDefaultValue(gradeRangeFilter)
-            return
-          }
-          setDefaultLabel(settingNotUpdated)
-        }).catch(err => {
-        setDefaultLabel(settingNotUpdated)
-      })
+  // if user setting is saved, don't show checkbox and sync userSavedFilterSetting with assignmentGradeFilter
+  useEffect(() => {
+    console.log()
+    if (userSettingSaved) {
+      setShowSaveSetting(false)
+      setUserSavedFilterSetting(resourceGradeFilter)
     }
-  }
+  }, [userSettingSaved])
 
   useEffect(() => {
+    console.log("Setting the slider range")
```

Comment:
Other than the `console.log()` looks good to me

Reply:
Done